custom-images/docker-compose.override.yml

```diff
@@ -28,16 +28,21 @@ services:
     volumes:
       - ${LOCAL_DATA_PATH}\mssql:c:\data
   
+  # Mount our Solr data folder and use our retagged Solr image.
   solr:
-    image: ${REGISTRY}${COMPOSE_PROJECT_NAME}-xp0-solr:${VERSION:-latest}
+    volumes:
+      - type: bind
+        source: .\docker\data\solr
+        target: c:\data
+
+  # Some modules (like SXA) also require additions to the Solr image.
+  solr-init:
+    image: ${REGISTRY}${COMPOSE_PROJECT_NAME}-xp0-solr-init:${VERSION:-latest}
     build:
-      context: ./docker/build/solr
+      context: ./docker/build/solr-init
       args:
-        BASE_IMAGE: ${SITECORE_DOCKER_REGISTRY}sitecore-xp0-solr:${SITECORE_VERSION}
+        BASE_IMAGE: ${SITECORE_DOCKER_REGISTRY}sitecore-xp0-solr-init:${SITECORE_VERSION}
         SXA_IMAGE: ${SITECORE_MODULE_REGISTRY}sxa-xp1-assets:${SXA_VERSION}
-    mem_limit: 1GB
-    volumes:
-      - ${LOCAL_DATA_PATH}\solr:c:\data
 
   id:
```

Comment:
Set SITECORE_DEVELOPMENT_PATCHES: CustomErrorsOff for 'id'? Noticed we're doing it for the other topologies.

Reply:
hmm should just be on CM and CD

Reply:
Ah yes ignore this, misread the diff :)